api/src/database/index.ts

```diff
@@ -29,6 +31,8 @@ sequelize.authenticate();
 
 export const DB = {
   Users: UserModel(sequelize),
+  // Why does this need the 'new' keyword?
```

Comment:
??

Reply:
oh my bad, I had an error previously, but I fixed it and I forgot to delete the comment